src/App.tsx

```diff
@@ -28,104 +52,136 @@ function clickedGoogle() {
 }
 
 function App() {
-
   const [selected, setSelected] = React.useState("");
 
   const storeNewDefault = (newDefault: string) => {
-    chrome.storage.local.set({selected: newDefault}, ()=>{});
-    setSelected(newDefault)
-  }
+    chrome.storage.local.set({ selected: newDefault }, () => {});
+    setSelected(newDefault);
+  };
 
   const getLastDefault = () => {
     try {
       chrome.storage.local.get("selected", ({ selected }) => {
-        setSelected(selected)
-      });   
-    } catch(err) {
-      console.log(err)
+        setSelected(selected);
+      });
+    } catch (err) {
+      console.log(err);
     }
-  }
+  };
 
   React.useEffect(() => {
-    getLastDefault()
-  }, [])
+    getLastDefault();
+  }, []);
 
   return (
     <Styles.App>
       <Styles.AppHeader>
         <Styles.YouLogoContainer
-          src={ `${CDN_YDC_BASE}/shared/logos/ydc-logo-lightdarkmode.svg` }
+          src={`${CDN_YDC_BASE}/shared/logos/ydc-logo-lightdarkmode.svg`}
           alt="Logo"
         />
-        <Styles.Title>Search Experience</Styles.Title>
-        <Styles.SubTitle>Search from the URL bar using</Styles.SubTitle>
+        <Styles.Title>Create with AI</Styles.Title>
+        <Styles.SubTitle>Generative AI tools to do more</Styles.SubTitle>
 
         <Styles.SuiteWrapper>
-
-          <SuiteComponent 
-            suiteTitle="You.com"
-            logo={<SearchLogo />}
-            selected={selected === YOUALL}
+          <SuiteComponent
+            suiteTitle="Write"
+            logo={<WriteSquareLogo />}
```

Comment:
Should we create an abstract svg element and move the actual svg to our CDN. This would help to maintain the logos without needing to publish a new extension version